source/common/res/features/popup-calculator/settings.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
The settings.js file wasn't handled correctly on Windows but I can't remember exactly why. I'll take another look at it this evening. I'll revert the file once I figure out why it wasn't working.